finagle-http/src/main/scala/com/twitter/finagle/http/Status.scala

```diff
@@ -29,7 +29,54 @@ case class Status(code: Int) {
     }
 }
 
+
 object Status {
+
+  /**
+   * Matches when the status code isn't in range of any of the  categories: {{Informational}},
+   * {{Successful}}, {{Redirection}}, {{ClientError}}, {{ServerError}}.
+   */
+  object UnknownStatus {
```

Comment:
How about to rename it just `Unknown`? We already have prefix `Status` so `Status.UnknownStats` feels redundant.